.github/workflows/ci.yml

```diff
@@ -56,18 +56,17 @@ jobs:
         run: bash ./script/git-version.sh --verbose get
 
       - name: Restore
-        run: dotnet msbuild build/build.proj -target:Restore
+        run: dotnet msbuild build/build.proj -target:Restore -v:d
 
       - name: Build
-        run: dotnet msbuild build/build.proj -target:Build -p:NoRestore=true
+        run: dotnet msbuild build/build.proj -target:Build -p:NoRestore=true -v:d
 
       - name: Test
-        run: dotnet msbuild build/build.proj -target:Test -p:NoRestore=true -p:VSTestNoBuild=true
+        run: dotnet msbuild build/build.proj -target:Test -p:NoRestore=true -p:VSTestNoBuild=true -v:d
 
   pack:
-    if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v')
     name: Create package
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
```

Comment:
Why change this?

Reply:
1. When not building for net461 it is not necessary to use windows
2. ubuntu is what was used before #34 and has documented successes in completing all of the tasks